Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -496,6 +496,75 @@ public static class Defaults
             }
         };
 
+        public static IList<AlarmDefinition> Alb = new List<AlarmDefinition>
+        {
+            new AlarmDefinition
+            {
+                Name = "Alb5xxErrorsHigh",
+                Metric = "HTTPCode_ELB_5XX_Count",
+                Period = TimeSpan.FromMinutes(1),
+                EvaluationPeriods = 1,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 10
+                },
+                DimensionNames = new[] {"LoadBalancer"},
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                Statistic = Statistic.Sum,
+                Namespace = AwsNamespace.Alb
+            },
+            new AlarmDefinition
+            {
+                Name = "AlbTarget5xxErrorsHigh",
+                Metric = "HTTPCode_Target_5XX_Count",
+                Period = TimeSpan.FromMinutes(1),
+                EvaluationPeriods = 1,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 10
+                },
+                DimensionNames = new[] {"LoadBalancer"},
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                Statistic = Statistic.Sum,
+                Namespace = AwsNamespace.Alb
+            },
+            new AlarmDefinition
+            {
+                Name = "AlbRejectedConnectionCountHigh",
+                Metric = "RejectedConnectionCount",
+                Period = TimeSpan.FromMinutes(1),
+                EvaluationPeriods = 1,
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 10
+                },
+                DimensionNames = new[] {"LoadBalancer"},
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                Statistic = Statistic.Sum,
+                Namespace = AwsNamespace.Alb
+            },
+            new AlarmDefinition
+            {
+                Name = "AlbTargetResponseTimeHigh",
+                Metric = "TargetResponseTime",
+                Period = TimeSpan.FromMinutes(1),
+                EvaluationPeriods = 1,
+                ExtendedStatistic = "p99",
+                Threshold = new Threshold
+                {
+                    ThresholdType = ThresholdType.Absolute,
+                    Value = 10
+                },
+                DimensionNames = new[] {"LoadBalancer"},
+                ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
+                Statistic = Statistic.Average,
```

Comment:
do we need to set this as well as ExtendedStatistic ? just seems a bit confusing

Reply:
Removed the `Statistic` as it is not needed when you specific `ExtendedStatistic`.